### PR TITLE
[AP-3851] Remove MTR phase one flag

### PR DESCRIPTION
**Pull request by naseberry**
## What

[Remove MTR phase one flag](https://dsdmoj.atlassian.net/browse/AP-3851)

MTR Phase one is released, we no longer require the code, this PR aims to tidy the codebase and remove unneeded code related to the feature flag.

## Checklist

- [x] revert initial feature flag setup (see: https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/4577)
- [x] remove dependant conditional code and logic such that MTR phase one is default
- [x] remove unit tests targeting this code and logic
- [x] remove feature tests targeting this code and logic
- [ ] other

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: features/providers/means_report.feature [hunk 1]
```diff
@@ -378,7 +378,6 @@ Feature: Means report
       | h3  | Bank statements |
 
   Scenario: For a non means tested journey
-    Given the feature flag for means_test_review_phase_one is enabled
     And I have completed a non means tested application
```
Comment (colinbruce) [inline features/providers/means_report.feature:381]: Nit: first step should start with `Given`

File: features/providers/non_means_tested_journey/certification_test.feature [hunk 2]
```diff
@@ -1,16 +1,7 @@
...
-    And I have created an application for a 17 year old
-    And I should be on the "confirm_client_declaration" page showing "Confirm the following"
-    Then I should see "they may have to pay towards legal aid"
-    And I should not see "their date of birth on the application is correct based on the information you have"
-
-  @javascript @vcr
-  Scenario: Completes an application to the certification page for applicant that is 17 with MTR phase 1 enabled
-    Given the feature flag for means_test_review_phase_one is enabled
+  Scenario: Completes an application to the certification page for applicant that is 17
     And I have created an application for a 17 year old
```
Comment (colinbruce) [inline features/providers/non_means_tested_journey/certification_test.feature:5]: Nit: first step should start with `Given`

File: features/providers/non_means_tested_journey/under_18_application_journey.feature [hunk 3]
```diff
@@ -56,4 +56,6 @@ Feature: Under 18 applicant journey with means test review phase one disabled
     Then I should be on a page with title "Check your answers"
 
     When I click 'Save and continue'
-    And I should be on a page showing "DWP records show that your client does not get a passporting benefit"
+    And I pause
```
Comment (colinbruce) [inline features/providers/non_means_tested_journey/under_18_application_journey.feature:59]: Pausing slows down tests, is this still needed?

File: features/providers/non_means_tested_journey/without_delegated_functions.feature [hunk 4]
```diff
@@ -1,8 +1,7 @@
 Feature: Non-means-tested applicant journey without use of delegation functions
 
   @javascript @vcr
-  Scenario: Completes an application for applicant that is under 18 with MTR phase 1 enabled
-    Given the feature flag for means_test_review_phase_one is enabled
+  Scenario: Completes an application for applicant that is under 18
     And I am logged in as a provider
```
Comment (colinbruce) [inline features/providers/non_means_tested_journey/without_delegated_functions.feature:5]: Nit: first step should start with `Given`

**Review by colinbruce — COMMENTED** [review]
Looking good, a few nits about features and a new pause which seems odd 😕 
Also, the coverage is failing because the CCMS for under age is not being hit which seems weird!

**Comment by naseberry** [comment]
> Looking good, a few nits about features and a new pause which seems odd 😕 Also, the coverage is failing because the CCMS for under age is not being hit which seems weird!

The ticket suggested we delete `applicant_under_16_blocked.feature`, doing this caused code coverage to fail.

**Review by colinbruce — APPROVED** [review]
LGTM :+1:

**Pull request merged**